docs/templating-guide/protocols/http.md

```diff
@@ -572,3 +572,51 @@ requests:
     threads: 5
     race: true
 ```
+
+**Requests Annotation**
+
+The recent nuclei version added the capability to perform per request properties/behavior override via request inline annotations. They are very similar to python/java class annotations and must be put on the request just before the RFC line. Currently, only the following overrides are supported:
```

Comment:
Since this is documentation, and not a blog post, instead of the "recent version", we should explicitly state the version in which this functionality was introduced or we should omit it altogether.

Reply:
intro has been generalized